projects/ids-enterprise-ng/src/lib/calendar/soho-calendar.d.ts

```diff
@@ -101,6 +101,14 @@ interface SohoCalendarRenderMonthEvent {
   year: number;
 }
 
+interface SohoCalendarWeekOptions {
```

Comment:
Noticed two missing options https://github.com/infor-design/enterprise/blob/master/src/components/calendar/calendar.js#L41-L42

Reply:
Adding   onChangeView and showToday